libtransmission/bitfield.cc

```diff
@@ -61,13 +60,12 @@ size_t Bitfield::countRangeImpl(size_t begin, size_t end) const
         return 0;
     }
 
-    if (first_byte >= this->alloc_count_)
+    if (first_byte >= this->bits_.size())
```

Comment:
I'd prefer `std::size(foo)` over `foo.size()` here and elsewhere

Reply:
Changed `.size()` to `std::size`